bin/systemd_module_migration.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+if ! which hub > /dev/null ; then
+  echo "We require hub later on for the PR" >&2
+  exit 1
+fi
+
+# fix all modules with systemd in .fixtures.yml
+sed --in-place 's#camptocamp/puppet-systemd#voxpupuli/puppet-systemd#' modules/voxpupuli/puppet-*/.fixtures.yml
```

Comment:
FreeBSD does not have long options… A more portable solution would be:

```suggestion
sed -I '' 's#camptocamp/puppet-systemd#voxpupuli/puppet-systemd#' modules/voxpupuli/puppet-*/.fixtures.yml
```

…but POSIX does not have in-place edit either (no `-I` / `-i`) so I am not sure about the best way to handle this if we want it to be portable.

Maybe just `set -e` at the beginning of the script to exit as soon as the first command fail?

Reply:
You've got ruby involved with a puppet agent so you could use that :)

```
ruby -pi -e "gsub(/before/, 'after')" filename
```